holoviews/plotting/plot.py

```diff
@@ -300,7 +300,14 @@ def selection_display(self, obj):
         return getattr(plt_class, 'selection_display', None)
 
     def _define_interface(self, plots, allow_mismatch):
-        parameters = [{k:v.precedence for k,v in plot.param.params().items()
+        def param_objects(obj):
+            if hasattr(obj.param, 'objects'):
+                return obj.param.objects()
+            else:
+                # Deprecated in Param 1.12.0
+                return obj.param.params()
```

Comment:
You haven't done this other than here.  

We should just pin to param 1.12.0 and use `xxx.param.objects()` everywhere. I can see Panel has it pinned to that version too.

Reply:
I was wondering why I did that here only and not everywhere, and then noticed that the other places were in some test files. If I remember correctly, my reasoning was that the tests are more likely to be run with the latest Param, while the code itself is going to be run with whatever version of Param users happen to have.

Weird reasoning maybe! And I should have indicated that in the comment to make the review easier for sure.

I'll bump the minimum required version of Param to 1.12.0.